spring-boot/frontend/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --experimental-https --experimental-https-cert=/code/certificates/localhost.pem --experimental-https-key=/code/certificates/localhost-key.pem",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@userfront/next": "^1.0.0",
```

Comment:
I think if you're only using `@userfront/next/client` you can drop-in `@userfront/react` instead.

Reply:
I'm going to leave it as userfront/next to lower confusion. I think its fine for developers to keep the mindset of "I'm using nextjs, I should import `@userfront/next`".